public_suffix_list.dat

```diff
@@ -1073,6 +1076,16 @@ edu.gh
 gov.gh
 org.gh
 mil.gh
+net.gh
```

Comment:
Please resubmit the changes with these all in alphabetical order, and place the appropriate _PSL TXT record in the zone for each that matches the PR # as in the instructions.